post-meta-inspector.php

```diff
@@ -93,7 +93,7 @@ public function post_meta_inspector() {
 		?>
 			<?php foreach( $values as $value ) : ?>
 			<?php
-				$value = var_export( $value, true );
+				$value = apply_filter( 'pmi_modify_post_meta_value', var_export( maybe_unserialize( $value ), true ), $key, $value );
```

Comment:
Two other things:
- The filter name should probably just be `pmi_post_meta_value`; we're not actually modifying the value so the name is misleading.
- Why are we unserializing now but weren't before?